tools/obscuroscan_v2/backend/webserver/webserver.go

```diff
@@ -24,6 +25,12 @@ func New(backend *backend.Backend, bindAddress string, logger log.Logger) *WebSe
 	r.RedirectTrailingSlash = false
 	gin.SetMode(gin.ReleaseMode)
 
+	config := cors.DefaultConfig()
+	config.AllowOrigins = []string{"*"}
```

Comment:
Should add a todo to do this properly with just the permissioned origin(s) probably, unless you're thinking you want this to be a public API service for people to build tools against.

Reply:
Prolly for now it's ok for people to hook up, but yes will add a todo to keep an eye on 👍